### DISCUSSION TITLE
Discussing Contents of Wiki

### DISCUSSION THREAD

**Discussion by WasteOfO2** [saber-notes/saber#471] [Ideas]
Refer https://github.com/adil192/saber/issues/438

Every project should have some sort of documentation/wiki for users or developers to refer to.

The benefits are numerous, and a well made wiki minimizes common issues raised by users, and provides a starting point for contributors.

This discussion is aimed at making a solid foundation for documentation of Saber. I propose the following sections:

0. Translate This Wiki
1. About Saber
2. Installing Saber
3. Using Saber
4. Saving Your Notes On Nextcloud
5. FAQs regarding the project
6. Common Issues and Troubleshooting
7. Available and Pending Translations
8. Contributing Guidelines (for Developers)
9. Contributing Guidelines (for Translators)
10. How to Donate to Saber
11. Roadmap/Future Plans/Planned Features (whatever title suits the best)

Each of these are proposed sections in a SPECIFIC order. Each section can contain multiple pages.

Using Markdown is advised.

Section 5, 8(especially this), 9 and 11 require directions from @adil192 . Rest can be handled by me and the community.

Please suggest your thoughts and opinions. Any changes to the order of sections or sections themselves are welcome.

Also discuss contents for each of the section. I will start working from potentially 1st or 2nd week of April as I have final exams.

**Comment by ceskyDJ** [comment]
Thanks for your activity on Saber's wiki creation! As you can clearly see, I am not much into English, so I'll wait a bit and after that I'll try to translate the work of others to Czech. I would like to be tagged, when some English part will be ready to translate to another languages.

My personal opinion: translating of developer's sections isn't too necessary. Maybe it should be completely split from users' sections (users don't need to see notes for developers, it is more something like internal thing). Developers usually speaks English, so there shouldn't be problem to let their part in English or just give a priority to part for users.

**Reply by WasteOfO2** [reply]
We can choose to not translate dev's section, but that would make the wiki a bit inconsistent

Although yes i can see why it shouldnt be translated

**Reply by ceskyDJ** [reply]
At least we should split user and developer sections, so user won't be disturbed by developer stuff. I think the main purpose of wiki will be for new users looking for available features and help with something they want to set up, etc.

**Reply by WasteOfO2** [reply]
Sure

**Comment by WasteOfO2** [comment]
I will need some time to start work on wiki. Exams have been announced and couldn't have come at a worse time. Please give me upto 2 months

**Reply by g14p** [reply]
Did you upload it somewhere by now?

**Comment by WasteOfO2** [comment]
I think we can merge section 7 and 9, thoughts?

**Reply by ceskyDJ** [reply]
That is very nice but where can I find the wiki you are working on?

**Reply by WasteOfO2** [reply]
I only currently have the wiki cloned and being worked on my PC, it is not available on any repo atm

**Reply by ceskyDJ** [reply]
Oh, ok. I thought you have noted sections completely done and see it should be merged into one. That's my bad, I'm sorry for misunderstanding. I think it should be available somewhere online, do others could see your progress and/or try to work on sections you don't work on. What do you think?

**Reply by ceskyDJ** [reply]
Sections 7 and 9 could be merged. But the contribution part should have a reminder it requires a little of "IT knowledge" (it's somewhere between user and developer part).

**Reply by WasteOfO2** [reply]
I can create a draft pull request and you can review what changes are being made. I could also just commit all what I have done now and mark other sections as WIP. Eitherway, I would have to make a repo anyways. I will do it soon

Sure, i will mention that part as well, we could make a tutorial for translating down the line, but i think it is best to atleast have the skeleton of the wiki ready

**Comment by WasteOfO2** [comment]
Initial changes have been made, I have not added a page for i18n contributions cuz so far it is incomplete, I have no issues adding that page as is, but it would not be a nice look.

I think it should set a decent standard for what format should be expected for the wiki, decorations like footers and sidebars can be made later

**Comment by g14p** [comment]
Where can I find it? I would love to help at some point and a wiki would be useful.

**Reply by g14p** [reply]
I found it i think! https://github.com/saber-notes/saber/wiki/
